buf.gen.yaml

```diff
@@ -1,7 +1,13 @@
 version: v1
 plugins:
-  - name: js
```

Comment:
This file should be in your android/ios/flutter type definition folders. Here are some helpful resources that should provide some clarity:
- [motor-flutter buf.build generation script](https://github.com/sonr-io/motor-flutter/blob/main/scripts/proto.sh)
- [motor-flutter buf.gen.yaml file](https://github.com/sonr-io/motor-flutter/blob/main/lib/buf.gen.yaml)
- [MotorKit buf.gen.yaml file](https://github.com/sonr-io/MotorKit/blob/main/Sources/MotorKit/buf.gen.yaml)